templates/download/thank-you.html

```diff
@@ -94,6 +94,7 @@ <h3 class="p-card--title p-heading--4">Ubuntuニュースレターを購読</h3>
 
           <input type="hidden" name="Consent_to_Processing__c" value="yes" />
           <input value="3649" name="formid" type="hidden">
+          <input type="hidden" name="returnURL" aria-label="returnURL" value="https://jp.ubuntu.com/thank-you">
           <input type="hidden" name="thankyoumessage" value="登録ありがとうございます。<br><a href='https://assets.ubuntu.com/v1/f401c3f4-Ubuntu_Server_CLI_pro_tips_2020-04.pdf' target='_blank' >これがCLIチートシートです。</a>">
```

Comment:
Looks like this one is pointing the user to a download. Therefore, the returnURL should at least be the asset

Reply:
Hey @anthonydillon, thanks fo your review! I tried to return to the asset but it didn't really work. It just kept showing an empty page 
<img width="653" alt="Screenshot 2022-01-19 at 11 51 07 am" src="https://user-images.githubusercontent.com/57550290/150132359-ddd1169d-4c4c-4796-940b-cdfdda12a16f.png">

so I added a query `?page=thank-you` to the path `/download` so it shows a thank-you message with the link for users to download when it has this `?page=thank-you` query.

https://jp-ubuntu-com-376.demos.haus/download (without the query) 
![image](https://user-images.githubusercontent.com/57550290/150132889-0814acb7-5ff7-4daf-9c31-b76b7743536f.png)

https://jp-ubuntu-com-376.demos.haus/download?page=thank-you (with the query)
<img width="1435" alt="Screenshot 2022-01-19 at 12 31 23 pm" src="https://user-images.githubusercontent.com/57550290/150133116-41d459dd-9367-4020-80b6-dae76f5fe804.png">

What do you think? If it doesn't look good, I can try different ways!